CIScripts/Test/Tests/vRouterAgentMSIInstaller.Tests.ps1

```diff
@@ -0,0 +1,32 @@
+Param (
+    [Parameter(Mandatory=$true)] [string] $TestbedAddr
+)
+
+. $PSScriptRoot\..\Utils\CommonTestCode.ps1
+. $PSScriptRoot\..\TestConfigurationUtils.ps1
+. $PSScriptRoot\..\..\Common\Aliases.ps1
+. $PSScriptRoot\..\..\Common\VMUtils.ps1
+
+. $PSScriptRoot\..\GetTestConfigurationCodiLegacy.ps1
+$TestConf = Get-TestConfiguration
+$Session = New-PSSession -ComputerName $TestbedAddr -Credential (Get-VMCreds)
+
+Describe "vRouter Agent MSI installer" {
+    Context "installation and uninstallation" {
+        It "registers/unregisters agent service and never enables" {
+            Install-Agent -Session $Session
```

Comment:
The Agent is installed by default, so we're not actually testing installation here. Maybe just copy-paste the body of this test twice?

Reply:
Or maybe reorder: first uninstall, then install

Reply:
We discussed it with @jablonskim - we won't install artifacts by default in "Provision" stage.